nailgun/entities.py

```diff
@@ -4943,6 +4922,16 @@ def manifest_history(self, synchronous=True, **kwargs):
         )
         return _handle_response(response, self._server_config, synchronous)
 
+    def read(self, entity=None, attrs=None, ignore=None):
+        """Ignore ``organization`` field as it's never returned by the server
```

Comment:
Intended. Subscription simply don't have such field at all. It was added to be able to store helpers like `manifest_upload` under Subscription, not Organization class (although technically it should had belonged to Organization).